packages/docs/docs/tutorial/part2.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 2
+---
+
+# Tutorial Part 2 - Progressive rendering
+
+Let's look at a slightly more complex example, which uses the `<Inline>` component
+to progressively render the result of the LLM invocation as it is generated. This
+allows us to take the current result generated by the LLM and pass it into the next
+invocation of the LLM.
+
+```tsx filename="packages/tutorial/src/part2.tsx"
+import * as LLMx from 'ai-jsx';
+import { Inline } from 'ai-jsx/core/inline';
+import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+const app = (
+  <Inline>
+    <ChatCompletion>
+      <UserMessage>Come up with the name of a mythical forest animal.</UserMessage>
+    </ChatCompletion>
+    {'\n\n'}
+    {(conversation) => (
```

Comment:
Maybe we should rename `conversation` to `mythicalAnimalName`, and the second `conversation` to `mythicalAnimalNameAndDescription`. That could make it clearer what the content will be.